src/client/tile.cpp

```diff
@@ -129,11 +129,12 @@ std::string getTexturePath(const std::string &filename)
 	/*
 		Check from texture_path
 	*/
-	const std::string &texture_path = g_settings->get("texture_path");
-	if (!texture_path.empty()) {
-		std::string testpath = texture_path + DIR_DELIM + filename;
+	for (const auto &path : fs::GetRecursiveDirs(g_settings->get("texture_path"))) {
```

Comment:
is this possible to cache fs::GetRecursiveDirs for this value ? it's called many times

Reply:
Possible. Where could be the variable defined? It is used both in the server and in the client.

Reply:
@nerzhul